tests/unit/package_managers/test_npm.py

```diff
@@ -605,14 +792,17 @@ def test_fetch_npm_source(
                 r["dependencies_to_download"],
             )
         )
-    mock_get_npm_dependencies.assert_has_calls(calls)
+    """
+    # mock_get_npm_dependencies.assert_has_calls(calls)
     expected_output = RequestOutput.from_obj_list(
         components=request_output["components"],
         environment_variables=request_output["environment_variables"],
         project_files=request_output["project_files"],
     )
 
     assert output == expected_output
+    # update_package_lock_with_local_paths.assert_called()
+    # update_package_json_files.assert_called()
```

Comment:
commented-out asserts can probably be removed?

Reply:
yeah sure, sry